src/Spago/Build.hs

```diff
@@ -63,6 +65,22 @@ build BuildOptions{..} maybePostBuild = do
   let allPsGlobs = Packages.getGlobs   deps depsOnly configSourcePaths <> sourcePaths
       allJsGlobs = Packages.getJsGlobs deps depsOnly configSourcePaths <> sourcePaths
 
+      checkImports globs = do
+        graph <- Purs.graph globs
+        case graph of
+          Left err -> die [ displayShow err ]
+          Right (ModuleGraph moduleGraph) -> do
+            let
+              getPackage = fmap PackageName . headMay . drop 1 . dropWhile (/= ".spago") . Text.split (== '/')
```

Comment:
I'm not sure if this is the best way to reify the package name from a module path - but it's what came to mind.

I have concerns particularly about whether or not the build could technically be run from a descendent of a directory `.spago`, in which case this check wouldn't work.  @f-f do you think this is a valid concern?